solidity/test/LiquidityProtectionEdgeCases.js

```diff
@@ -44,7 +44,7 @@ const CONFIGURATIONS = [
 ];
 
 const NUM_OF_DAYS = [30, 100];
-const DECIMAL_COMBINATIONS = cartesian([12, 25], [12, 25], [15, 22], [15, 22]);
+const DECIMAL_COMBINATIONS = cartesian([12, 24], [12, 24], [15, 21], [15, 21]);
```

Comment:
@barakman can you also add the edge case of 6 decimals (USDT, USDC) trading against 18 decimals (BNT)?

Reply:
I can, but it's gonna be a little complicated, because I'd still need to avoid the combination of 6 and 24, which will surely not work.

Reply:
You can add an `if` to skip this combination or to push it explicitly to `DECIMAL_COMBINATIONS` afterwards

Reply:
any of these options work

Reply:
This requires a more sophisticated (or at least a different) test.
Just changing the numbers from "12,24" to "6,18" is not going to achieve a lot.

At present, this test does the following:
- Add an initial amount of 10^12 or 10^24 BNT and 10^12 or 10^24 TKN
- Protect an additional amount of 10^15 or 10^21 BNT and 10^15 or 10^21 TKN
- Increase the rate of BNT or TKN or neither (via converting in one direction)
- Generate a fee of BNT or TKN or neither (via converting back and forth)
- Wait for 30 or 100 days
- Call `removeLiquidityReturn` and make sure that the result makes sense

The level of extremeness is generally determined by **the ratio** between the amounts.
So reducing each one of them by a magnitude of 10^6 is not going to make a big impact.